src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationevents/repository/model/data/EventNotification.kt

```diff
@@ -35,4 +35,19 @@ class EventNotification(
   @Temporal(value = TemporalType.TIMESTAMP)
   @Column(name = "LAST_MODIFIED_DATETIME", nullable = false)
   val lastModifiedDateTime: LocalDateTime,
-)
+) {
+  override fun equals(other: Any?): Boolean {
+    if (other == null) return false
```

Comment:
What happens if `this` is null?

Reply:
The equals method isn't a static method, so surely `this `can't be null for equals to be called?